opentechinstitute/luci-commotion
# Updating commotion-debug-helper to not use iwinfo or luci_splash

To test:
1. Run commotion-debug-helper through the luci interface, ensure that the text file that you download contains debug output and doesn't have any blank sections or sections that only include errors. In this context, a section is a header:

-----------------------------------------------------------------
<command description>
-----------------------------------------------------------------
<output>

So <output> should not be blank and should not just say "command not found" or something.
2. Login to the node over SSH and run 'cdh all'. Check /tmp/debug.info as above.
